mnefun/_mnefun.py

```diff
@@ -1792,19 +1792,20 @@ def _viz_raw_ssp_events(p, subj, ridx):
                 if ri in p.get_projs_from]
     all_proj = op.join(pca_dir, 'preproc_all-proj.fif')
     projs = read_proj(all_proj)
-    ev_names = [op.join(pca_dir + '/' + ii) for ii in
-                ['preproc_ecg-eve.fif', 'preproc_blink-eve.fif']]
+    ev_names = glob.glob(op.join(pca_dir, 'preproc*-eve.fif'))
     ev = [read_events(e) for e in ev_names]
-    assert len(ev) == 2
-    ev = np.concatenate((ev[0], ev[1]))
-    ev = ev[np.argsort(ev[:, 0], axis=0)]
+    if len(ev) == 2:
+        ev = np.concatenate((ev[0], ev[1]))
+        ev = ev[np.argsort(ev[:, 0], axis=0)]
+        colors = {int(idd): c for c, idd in zip(['r', 'b'], np.unique(ev[:, 2]))}
+    else:
+        ev = ev[0]
+        colors = {int(np.unique(ev[:, 2])[0]): 'r'}
     raw = _raw_LRFCP(pre_list, p.proj_sfreq, None, None, p.n_jobs_fir,
                      p.n_jobs_resample, projs, None, p.disp_files,
                      method='fft', filter_length=p.filter_length,
                      force_bads=False)
-    raw.plot(events=ev, event_color={999: 'r', 998: 'b'})
-    plt.draw()
-    plt.show()
```

Comment:
Does the plotting in your IDE work if you put this `plt.draw()` or `plt.show()` back in?